drivers/ethernet/eth_adin2111.c

```diff
@@ -351,13 +351,24 @@ static int eth_adin2111_send_oa_frame(const struct device *dev, struct net_pkt *
 	uint16_t clen, len = net_pkt_get_len(pkt);
 	uint32_t hdr;
 	uint8_t chunks, i;
-	int ret, txc, cur;
+	int ret, txc, cur, ebo;
 
 	chunks = len / ctx->oa_cps;
 
 	if (len % ctx->oa_cps) {
 		chunks++;
 	}
+	
+	if (chunks > 1) {
+		//we have to calculate EBO so we do not exceed maximum Ethernet frame length
```

Comment:
Do not use `//` comment. compliance will fail.

Reply:
FIxed